### PR TITLE
Run Python tests on CI

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This PR allows running the Python tests on CI. For that, it is required the following:
* Change workflow setup to run Python tests with `pytest`
* Bump Python dependencies
* Fix some Python tests (e.g. brittle logic)
* Fix doc according to the changes in the Python tests (e.g. test names and lines)

### Motivation and Context
The objective is to execute all test examples in CI. To that aim, this  PR is a continuation of #1081 (Java).

### Types of changes

- [X] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [X] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by diemol — APPROVED** [review]
Thank you, @bonigarcia!

File: examples/python/tests/getting_started/test_first_script.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
     title = driver.title
-    assert title == "Google"
+    assert "DuckDuckGo" in title
 
-    driver.implicitly_wait(0.5)
+    driver.implicitly_wait(10)
 
     search_box = driver.find_element(by=By.NAME, value="q")
-    search_button = driver.find_element(by=By.NAME, value="btnK")
+    search_button = driver.find_element(by=By.ID, value="search_button_homepage")
```
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:18]: ubmit_button = driver.find_element(by=By.XPATH, value="/html/body/main/div/form/div/div[2]/button")

File: examples/python/tests/getting_started/test_first_script.py [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+    driver.get("https://duckduckgo.com/")
 
     title = driver.title
-    assert title == "Google"
+    assert "DuckDuckGo" in title
 
-    driver.implicitly_wait(0.5)
+    driver.implicitly_wait(10)
 
     search_box = driver.find_element(by=By.NAME, value="q")
```
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:17]: text_box = driver.find_element(by=By.NAME, value="my-text")

File: examples/python/tests/getting_started/test_first_script.py [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
 
 
 def test_eight_components():
-    driver = webdriver.Chrome()
+    driver = webdriver.Chrome(service=ChromeService(executable_path=ChromeDriverManager().install()))
 
-    driver.get("https://google.com")
+    driver.get("https://duckduckgo.com/")
```
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:10]: driver.get("https://www.selenium.dev/selenium/web/web-form.html")

File: examples/python/tests/getting_started/test_first_script.py [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
 
     title = driver.title
-    assert title == "Google"
+    assert "DuckDuckGo" in title
```
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:13]: assert title == "Web form"

File: examples/python/tests/getting_started/test_first_script.py (hunk 2 above)
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:17]: text_box = driver.find_element(by=By.NAME, value="my-text")

File: examples/python/tests/getting_started/test_first_script.py (hunk 1 above)
Comment (leilachebbah) [inline examples/python/tests/getting_started/test_first_script.py:18]: submit_button = driver.find_element(by=By.XPATH, value="/html/body/main/div/form/div/div[2]/button")

**Pull request merged**